codeception/codeception/5.0/codeception.yml

```diff
@@ -0,0 +1,19 @@
+namespace: App\Tests
```

Comment:
IIRC codeception itself uses `Tests` as default namespace (without `App\`).

Should the recipe use the same default namespace?

Reply:
Hi @W0rma !

It was already discussed at https://github.com/symfony/recipes-contrib/pull/1432 , please read that thread.
In short, `Tests` is a **_should_**, not a _**must**_. If you find a way to use `Tests` without the user getting errors the first time he tries to run `vendor/bin/codecept run` you are welcome to modify the recipe.

Reply:
I see, thank you for the clarification!